ciradu2204/NaturalDisasters-Tracker
# Add readme

This PR updates the readme with content to help people understand the project purpose and help team members start contributing to the project.
## Changes 
- [x] Simple overview of use/purpose.
- [x] An in-depth paragraph about the project and overview of use.
- [x] Getting started: how to contribute
- [x] Contributors name and contact info
- [x] Acknowledgement

## Screenshots 
<img width="940" alt="Screen Shot 2021-06-10 at 09 58 43" src="https://user-images.githubusercontent.com/37863089/121487692-9e3c5000-c9d2-11eb-92fa-0af12c55e00f.png">

 ## Checklist
- [x] Looks good on large screens
- [x] Looks good on mobile

## How to test your code

1. From your terminal, pull down this branch with `git pull origin add-readme` and check that branch out with `git checkout add-readme`.
2. Check the readme file in the repository and see the new changes I have added

Fixes #1
